helm/alfresco-process-application/values.yaml

```diff
@@ -384,7 +398,7 @@ rabbitmq:
       nginx.ingress.kubernetes.io/x-forwarded-prefix: "true"
 
 persistence:
-  enabled: false
```

Comment:
i think false should be the default for now, then you enable it at runtime
via `helm install --set persistence.enabled=true ...`